src/cirrus/plugins/publishers/documentation.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""
+_documentation_
+
+Publisher plugin that uses fabric to do a remote put
+
+"""
+from fabric.operations import put
+from cirrus.fabric_helpers import FabricHelper
+from cirrus.logger import get_logger
+from cirrus.publish_plugins import Publisher
+
+
+LOGGER = get_logger()
+
+
+class Documentation(Publisher):
+    PLUGGAGE_OBJECT_NAME = 'file_server'
+
+    def publish(self, opts, doc_artifact):
+        """
+        publish docs to a remote server via fabric over ssh.
+
+        .. note:: This only uploads the doc artifact to the server. If
+            desired, the artifact must be manually unpacked on the server.
+
+        Requires the following cirrus.conf sections:
+        [doc]
+        publisher = file_server
+
+        [file_server]
```

Comment:
maybe name this section doc_file_server so that it is more obvious what it is for?